pkg/moov/transfer_models.go

```diff
@@ -126,6 +128,26 @@ type Transfer struct {
 	SalesTaxAmount *Amount `json:"salesTaxAmount,omitempty"`
 }
 
+// Fees charged to accounts involved in the transfer.
+type MoovFees struct {
```

Comment:
Noticed 1 more typo:
```suggestion
type MoovFee struct {
```